.ycm_extra_conf.py

```diff
@@ -5,13 +5,13 @@
 
 CONDA_PREFIX = os.environ["CONDA_PREFIX"]
 NVJITLINK_DIR = Path(__file__).parent
-VER = sys.version_info
-PYTHON_INCLUDE_NAME = "python3.10"  # Not ideal
+PYTHON_VERSION = sys.version_info
+PYTHON_INCLUDE_NAME = "python%d.%d" % PYTHON_VERSION
 
 CONDA_INCLUDE_DIR = Path(CONDA_PREFIX, "include")
 PYTHON_INCLUDE_DIR = Path(CONDA_INCLUDE_DIR, PYTHON_INCLUDE_NAME)
 NVJITLINK_INCLUDE_DIR = Path(NVJITLINK_DIR, "include")
-CUDA_INCLUDE_DIR = "/usr/local/cuda-12.0/include"
+CUDA_INCLUDE_DIR = "/usr/local/cuda/include"
```

Comment:
Actually in the Conda case wonder if this is obviated by having the new CTK packages. Would be interested to know

Reply:
My use cases often include using non-released CUDA toolkits that I have installed in `/usr/local`, so it'd be preferable to stick with the `/usr/local/cuda/include` "default".